src/fetch.js

```diff
@@ -1,8 +1,17 @@
+import domUpdates from './domUpdates';
+
 let apiFetch = {
+  confirmStatus(response) {
```

Comment:
This piece of code is vital for catching ALL errors, awesome way to catch these. Two thumbs way up!